os/fs/tmpfs/fs_tmpfs.h

```diff
@@ -75,6 +75,17 @@
 
 #define TFO_FLAG_UNLINKED (1 << 0)  /* Bit 0: File is unlinked */
 
+/* Redefine memory alloc function when using multi heap */
+
+#if CONFIG_MM_NHEAPS > 1 && CONFIG_MM_REGIONS > 1
+#undef		kmm_malloc
```

Comment:
Please correct the formatting

Reply:
Hello @drashti304 .
Could you let me know what kind formatting do you mean?

Reply:
I think the tab after #undef is not required.
in file view it looks like below:
#undef         kmm_realloc
#define  kmm_malloc(s)

Reply:
I think the tab after #undef is not required.
in file view it looks like below:
#undef \t kmm_realloc
#define kmm_malloc(s) malloc_at(CONFIG_FS_TMPFS_HEAP_INDEX, s)

Reply:
I think the tab after #undef is not required.
in file view it looks like below:
![formatting](https://user-images.githubusercontent.com/44993522/54663872-b4674880-4b08-11e9-8772-1256f1c1fdfd.PNG)

Reply:
I will modify